app/views/songs/index.html.erb

```diff
@@ -39,7 +39,9 @@
       serverSide: true,
       responsive: true,
       lengthChange: false,
-      dom: '',
+      pageLength: 25,
```

Comment:
I introduced pagination since there are lot of records after the DB is populated